events/hfe_events_discovery.txt

```diff
@@ -142,12 +171,206 @@ country_event = {
 				}
 				assign_leader = last_created_leader
 				set_location = owner.capital_star
+				save_global_event_target_as = hfe_discovery_fleet
 			}
 		}
 		add_resource = {
 			minerals = 500
 			energy = 500
 		}
+		country_event = {
+			id = hfe_discovery.2
+			days = 3
+		}
+		country_event = {
+			id = hfe_discovery.3
+			days = 10
+		}
+		country_event = {
+			id = hfe_discovery.4
+			days = 17
+		}
+		country_event = {
+			id = hfe_discovery.5
+			days = 24
+		}
+	}
+}
+
+country_event = {
+	id = hfe_discovery.2
+	title = HFE_DISCOVERY_MET_AMOEBAS
+	desc = HFE_DISCOVERY_MET_AMOEBAS_DESC
+	picture = GFX_evt_friendly_amoeba
+	location = event_target:hfe_discovery_fleet
+	show_sound = event_space_amoeba
+	is_triggered_only = yes
+#	show_sound = event_sensor_ping
+	trackable = no
+
+	immediate = {
+		every_country = {
+			limit = {
+				from = {
+					is_country_type = amoeba
+				}
+			}
+			save_global_event_target_as = hfe_call_bubbles
+			establish_communications = ROOT # ????????? ???????
+			# ????????? ???? ??????????????
+			set_country_flag = amoeba_encountered
+		}
+	}
+
+	option = { # Fight the amoebas
+		name = "HFE_DISCOVERY_AMOEBA_RESEARCH"
+		ROOT = {
+			enable_special_project = {
+				name = "AMOEBAS_PROJECT"
+				# The Vanilla rendition of this targets the planetary capital
```

Comment:
Wonder if it work

Reply:
Yes it does! I was very proud of these three Events, which fixed bugs about meeting the Space Creatures without their Events firing.